src/panels/config/integrations/integration-panels/zha/dialog-zha-change-channel.ts

```diff
@@ -90,7 +103,7 @@ class DialogZHAChangeChannel extends LitElement implements HassDialog {
             ${VALID_CHANNELS.map(
               (newChannel) =>
                 html`<ha-list-item .value=${String(newChannel)}
-                  >${newChannel}</ha-list-item
+                  >${this.hass.localize(`ui.panel.config.zha.change_channel_dialog.channel_${newChannel}`)}</ha-list-item
```

Comment:
Make a check here that checks is newChannel is auto. Then use translation otherwise just print the channel number. Adding 15 translations with just numbers will just give the translators more work.